README.md

```diff
@@ -3,26 +3,43 @@
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/232a8tadrrn8jv0k/branch/master?svg=true)](https://ci.appveyor.com/project/cgutman/moonlight-android/branch/master)
 [![Translation Status](https://hosted.weblate.org/widgets/moonlight/-/moonlight-android/svg-badge.svg)](https://hosted.weblate.org/projects/moonlight/moonlight-android/)
 
-[Moonlight for Android](https://moonlight-stream.org) is an open source client for NVIDIA GameStream and [Sunshine](https://github.com/LizardByte/Sunshine).
+[Moonlight](https://moonlight-stream.org/) for Android is an open source client for
+[Nvidia GameStream](https://www.nvidia.com/en-us/support/gamestream/) and
```

Comment:
Maybe not, but I figured since people still get support in `#geforce-experience` in Discord that maybe we weren't ready to stop mentioning it. I defer to the maintainer.